plugins/org.jkiss.dbeaver.ext.clickhouse/plugin.xml

```diff
@@ -131,6 +131,8 @@
             <type name="UUID"/>
             <type name="IPV4"/>
             <type name="IPV6"/>
+            <type name="TUPLE" includesTypedParams="true"/>
```

Comment:
We already support something similar in another place:
https://github.com/dbeaver/dbeaver/blob/d2c99a4ed519f9ea2cd818435408d6e36c6c28b6/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/registry/data/hints/AbstractValueBindingDescriptor.java#L168-L177
Is overcomplication worth it, or can we follow the `*` pattern approach instead?